packages/types/src/interfaces/xcm/definitions.ts

```diff
@@ -80,6 +80,20 @@ const location = {
       }
     }
   },
+  InteriorMultiLocation: 'Junctions',
+  Junctions: {
+    _enum: {
+      Here: 'Null',
+      X1: 'Junction',
+      X2: '(Junction, Junction)',
+      X3: '(Junction, Junction, Junction)',
+      X4: '(Junction, Junction, Junction, Junction)',
+      X5: '(Junction, Junction, Junction, Junction, Junction)',
+      X6: '(Junction, Junction, Junction, Junction, Junction, Junction)',
+      X7: '(Junction, Junction, Junction, Junction, Junction, Junction, Junction)',
+      X8: '(Junction, Junction, Junction, Junction, Junction, Junction, Junction, Junction)'
+    }
+  },
   Junction: {
```

Comment:
The `Junction` type needs to be versioned as well AFAICT.
The [`Parent` variant](https://github.com/paritytech/polkadot/blob/730c769505106d180f0febed3f050349d4502c20/xcm/src/v0/junction.rs#L110)
was [removed in v1](https://github.com/paritytech/polkadot/blob/730c769505106d180f0febed3f050349d4502c20/xcm/src/v1/junction.rs#L29)

Reply:
Dangit, was hoping to catch them all by now :(

Reply:
pr: https://github.com/polkadot-js/api/pull/3932